company-profiles/company-profiles/kyndryl.md

```diff
@@ -0,0 +1,128 @@
+# Kyndryl
+
+## Company blurb
+
+Kyndryl is a technology company that was spun off from IBM's infrastructure services business in 2021. It provides IT infrastructure services such as cloud, applications, data and AI, enterprise and zCloud, digital workplace, network and edge, and security and resiliency to enterprises. The company operates in 63 countries and has over 400 data centers. Kyndryl has formed partnerships with Microsoft and Google Cloud to provide digital transformation services to customers and help them become more data-driven and sustainable.
+
+## Company size
+
+90,000 Employees
+
+## Remote status
+
+5 days work week with flexible hours and WFH policy.
+
+## Region
+
+Worldwide
+
+## Company technologies
```

Comment:
This list could be smaller as its long to scroll down, maybe just list the ain technologies or place them into categories?